src/main/java/org/broadinstitute/hellbender/tools/exome/TangentNormalizer.java

```diff
@@ -200,7 +200,7 @@ public static TangentNormalizationResult tangentNormalizeNormalsInPoN(final PoN
         final List<String> sampleNamesCopy = new ArrayList<>(pon.getSampleNames());
 
         logger.info("Loading normalized counts...");
-        final ReadCountCollection coverageProfile = new ReadCountCollection(SetUniqueList.setUniqueList(pon.getTargets()), SetUniqueList.setUniqueList(sampleNamesCopy), pon.getNormalizedCounts());
+        final ReadCountCollection coverageProfile = new ReadCountCollection(pon.getTargets(), sampleNamesCopy, pon.getNormalizedCounts());
```

Comment:
The previous code was kind of clumsy in that there was no inherent reason for `SetUniqueList`.  It was *only* used as an indirect way to check for duplicates.  I have replaced this behavior with a more natural and easier-to-debug approach of throwing an error in the constructor if there are duplicates.